src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj

```diff
@@ -20,7 +20,11 @@
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <IncludeSymbols>true</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>	
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)'=='net8.0-windows10.0.19041'">
```

Comment:
Is this also needed for .NET 7?

Reply:
It's a breaking change from net8 see :

RuntimeIdentifiers have a new RID format see: [rid-graph](https://learn.microsoft.com/en-gb/dotnet/core/compatibility/sdk/8.0/rid-graph)
ex: win10-x86 is now win-x86